src/raftstore/store/peer.rs

```diff
@@ -1695,6 +1698,23 @@ fn get_transfer_leader_cmd(msg: &RaftCmdRequest) -> Option<&TransferLeaderReques
     Some(req.get_transfer_leader())
 }
 
+fn get_sync_log_from_request(msg: &RaftCmdRequest) -> bool {
+    if msg.has_admin_request() {
+        let req = msg.get_admin_request();
+        if req.get_cmd_type() == AdminCmdType::ChangePeer ||
```

Comment:
Why not `return req.get_cmd_type() == AdminCmdType::ChangePeer || req.get_cmd_type() == AdminCmdType::Split`?